scripts/markdown-references.mjs

```diff
@@ -0,0 +1,29 @@
+import path from "node:path";
+import fsp from "node:fs/promises";
```

Comment:
These imports could be simplified I guess?

```suggestion
import path from "path";
import fsp from "fs/promises";
```

Reply:
personally, im fine with it, indicates that these are built ins